server/src/test/java/org/elasticsearch/health/HealthServiceTests.java

```diff
@@ -141,59 +115,24 @@ public void testPreflightIndicatorResultsPresent() {
         );
 
         // Get all indicators returns preflight result mixed in with appropriate component
-        List<HealthComponentResult> health = service.getHealth(null, null, false);
-        assertThat(health.size(), is(equalTo(2)));
-        {
-            HealthComponentResult component1 = health.stream()
-                .filter(result -> result.name().equals("coordination"))
-                .findAny()
-                .orElseThrow();
-            assertThat(component1.status(), is(equalTo(YELLOW)));
-            assertThat(component1.indicators(), is(notNullValue()));
-            assertThat(component1.indicators(), containsInAnyOrder(hasMaster, networkLatency, slowTasks));
-        }
-        {
-            HealthComponentResult component2 = health.stream().filter(result -> result.name().equals("data")).findAny().orElseThrow();
-            assertThat(component2.status(), is(equalTo(GREEN)));
-            assertThat(component2.indicators(), is(notNullValue()));
-            assertThat(component2.indicators(), contains(shardsAvailable));
-        }
-
-        // Getting single component returns preflight result mixed in with appropriate component
-        health = service.getHealth("coordination", null, false);
-        assertThat(health.size(), is(equalTo(1)));
+        List<HealthIndicatorResult> health = service.getHealth(null, false);
+        assertThat(health.size(), is(equalTo(4)));
         {
```

Comment:
Nit: Do you think we should keep this and the blocks bellow? I am not sure if they are adding anything. I am referring only to the one liners.